e2e/cypress/fixtures/hooks/message_menus_with_datasource.json

```diff
@@ -0,0 +1,17 @@
+{
+    "attachments": [{
+        "pretext": "This is the attachment pretext.",
+        "text": "This is the attachment text.",
+        "actions": [{
+            "name": "Select an option...",
+            "integration": {
+                "url": "http://localhost:3000/message_menus_datasource",
+                "context": {
+                    "action": "do_something"
+                }
+            },
+            "type": "select",
+            "data_source": "channels"
+        }]
+    }]
+}
```

Comment:
Above payloads are the same as the ones we have at https://docs.mattermost.com/developer/interactive-messages.html#message-menus